fusequery/query/src/servers/clickhouse/clickhouse_stream.rs

```diff
@@ -115,6 +122,14 @@ impl Stream for ClickHouseStream {
     type Item = Result<ClickHouseBlock>;
 
     fn poll_next(mut self: Pin<&mut Self>, ctx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
+        // Some drivers will skip the first block for it recognizes the first block as schema
+        // So we make the first block to be an empty block
+        if self.block_index == 0 {
+            self.block_index += 1;
+            let block = DataBlock::empty_with_schema(self.schema.clone());
+            return Poll::Ready(Some(self.convert_block(block)));
+        }
```

Comment:
This block seems the root cause of https://github.com/datafuselabs/datafuse/issues/703.

What is its background? We don't always return schema as the title, at least in explain situation.

Reply:
Oh, it's just to be compatible with `clickhouse-rs`,  https://github.com/suharev7/clickhouse-rs/blob/async-await/src/types/query_result/stream_blocks.rs#L85

I'll take a look at it.